multiversx_sdk/network_providers/transaction_decoder.py

```diff
@@ -15,6 +15,10 @@ def __init__(self) -> None:
         self.function_name: Optional[str] = None
         self.function_args: Optional[list[str]] = None
         self.transfers: Optional[list[TokenTransfer]] = None
+        self.message: Optional[list[str]] = None
```

Comment:
If list, then plural. Also, can be `transfer_messages`, maybe? I think `bytes`, instead of `str`? Also, can be non-optional. Empty is not applicable.

Reply:
renamed and changed typed to bytes.